drivers/led_strip/apa102.c

```diff
@@ -94,6 +112,7 @@ static int apa102_init(const struct device *dev)
 
 static DEVICE_API(led_strip, apa102_api) = {
 	.update_rgb = apa102_update_rgb,
+	.update_channels = apa102_update_channels,
```

Comment:
It would be nice to provide a header to users with the channel mapping and also demonstrate this function from a sample.

Reply:
@simonguinot I updated the header

regarding the sample, I won't be able to test locally so let me know if you still want one.

would it make sense to update the `led_strip` (`zephyr/samples/drivers/led/led_strip`) so that it calls both `update_rgb` and `update_channels`?

Reply:
@simonguinot adding to that: I won't spend more time on contributing to the apa102 are we are no longer using those LEDs internally.

so I guess it's either we use that PR or we forget about it 😕 sorry about that

Reply:
> would it make sense to update the `led_strip` (`zephyr/samples/drivers/led/led_strip`) so that it calls both `update_rgb` and `update_channels`?

Yes it would but it is not really easy to have a generic/common `update_channels` test with different channel mapping on each controller.

Reply:
> @simonguinot adding to that: I won't spend more time on contributing to the apa102 as we are no longer using those LEDs internally.
> 
> so I guess it's either we use that PR or we forget about it 😕 sorry about that

Well it was a bit unfair from me to ask you to solve the sample thing. So yes it could OK.

Reply:
> @simonguinot I updated the header

I can't see any header. git add ?